examples/maps/maps.bal

```diff
@@ -43,6 +43,8 @@ public function main() {
 
     // Note that Index-based access will return a union of constraint+nil.
     // If the key does not exist a nil value is returned.
```

Comment:
Not related to this PR, but better if this is rephrased to say `nil` is returned instead of a `nil` value is returned.